hibernate-core/src/main/java/org/hibernate/SessionFactory.java

```diff
@@ -70,7 +70,7 @@
 	 * for use.
 	 * <p/>
 	 * Note that for backwards compatibility, if a {@link org.hibernate.context.spi.CurrentSessionContext}
-	 * is not configured but JTA is configured this will default to the {@link org.hibernate.context.internal.JTASessionContext}
+	 * is not configured but JTA is configured this will default to the {@code org.hibernate.context.internal.JTASessionContext}
```

Comment:
`internal` is excluded from javadoc, so I replace `@link` with `@code`. Let me know if you have better idea.

Reply:
Seems fine. Although we might want to ignore this warning, because one can decide to generate javadoc including the internal package I believe.

Reply:
Yeah, developers like you and me knows how to generate, but the main concern is the public user who looks up in publicly hosted Javadocs to look up some tech details.

Reply:
But then it already falls back to `@code`, it just generates the warning right?

Reply:
Yeah, I checked the public Javadoc and found it simply display the link text, so you are right.
The gist of this PR is to eliminate all possible Javadoc warning (most of them are because of misuse of Javadoc and therefore are detrimental to users). If it were possible, I would disable this warning for it seems this warning is not harmful like others. But it seems no such possibility exists?

Reply:
Yeah, I'm torn on this one

Reply:
It seems the only way to disable specific Javadoc warning is `@SuppressWarnings("javadoc")`, but it will hide any potential warnings.